docs/guides/unbound.md

```diff
@@ -157,3 +157,43 @@ Finally, configure Pi-hole to use your recursive DNS server by specifying `127.0
 ![Upstream DNS Servers Configuration](../images/RecursiveResolver.png)
 
 (don't forget to hit Return or click on `Save`)
+
+### Add logging to unbound
+
+!!! warning
+    It's not recommonded to increase verbosity for daily use, as unbound logs a lot. But it might be helpful for debugging purposes.
+
+There are five levels of verbosity
+
+```
+Level 0 means no verbosity, only errors
+Level 1 gives operational information
+Level 2 gives  detailed operational  information
+Level 3 gives query level information
+Level 4 gives  algorithm  level  information
+Level 5 logs client identification for cache misses
+```
+
+First, specify the log file and the verbosity level in the `server` part of
+`/etc/unbound/unbound.conf.d/pi-hole.conf`:
+
+```ini
+server:
+    # If no logfile is specified, syslog is used
+    logfile: "/var/log/unbound/unbound.log"
+    verbosity: 1
+```
+
+Second, create log dir and file, set permissions:
+
+```
+sudo mkdir /var/log/unbound
+sudo touch /var/log/unbound/unbound.log
+sudo chown unbound /var/log/unbound/unbound.log
```

Comment:
Check these commands for two conditions:

1. Path does not exist: (Hint: may need `mkdir -p`)
2. Path exists / file exists: (Does `mkdir` throw an error? Is `touch` silent?)

Reply:
`mkdir -p` solves both conditions: if dir exists, it doesn't return an error. If intermediate directories are missing, these will be created. 
https://stackoverflow.com/questions/22737933/mkdirs-p-option

`touch` is silent if file already exist, but will update timestamps
https://unix.stackexchange.com/questions/427237/what-happens-if-we-use-touch-command-while-the-file-already-exists